scripts/qa_crowbarsetup.sh

```diff
@@ -1863,7 +1867,7 @@ function onadmin_allocate
     else
         if [[ $hacloud = 1 ]] ; then
             cluster_node_assignment
-            local nodes=($unclustered_nodes)
+            local nodes=(${unclustered_nodes[@]})
```

Comment:
@aspiers Yes, it's stunning how many ways you can screw up in bash. And those ways are all subtle...